extensions/azure-functions-http/runtime/src/main/java/org/jboss/logging/AzureFunctionsLogger.java

```diff
@@ -0,0 +1,101 @@
+package org.jboss.logging;
```

Comment:
Small nitpick but please don't do that (same for the other class). It leads to split packages which causes all sorts of CL issues.

Keep this one in the `io.quarkus.azure.functions.resteasy.runtime` package. You can introduce a `logging` subpackage if you want.

That doesn't help you with your architecture questions though :).

Reply:
@gsmet totally agree with you, and that was what I originally coded, but I quickly discovered those Provider and Logger class heavily depend on package private objects from jboss-logging library.

Should I copy/paste as much code as needed instead? (which is lot...) I could not find another way to have a sort of sink for all log messages in the jboss-logging library (as we would do a LogHandler in log4j).

Reply:
I don't know. Both options look equally messy :).

Maybe @jamezp would have a clever idea?